Application Actions/Spotify/1.0/default.json

```diff
@@ -1,11 +1,7 @@
 {
-  "ButtonPress": "PlayPause",
+  "ButtonPress": "Play or Pause",
   "SwipeLeft":   "Previous Track",
   "SwipeRight":  "Next Track",
-  "SwipeUp":     "",
-  "SwipeDown":   "",
-  "RotateLeft":  "",
-  "RotateRight": "",
```

Comment:
Please squash both commits. Use `git rebase --interactive HEAD^^` and replace the second `edit` with `squash`. Save and `git push -f`